crates/proto/src/udp/udp_stream.rs

```diff
@@ -170,7 +170,10 @@ impl<S: UdpSocket + Send + 'static> Stream for UdpStream<S> {
             //   meaning that sending will be prefered over receiving...
 
             // TODO: shouldn't this return the error to send to the sender?
-            ready!(socket.poll_send_to(cx, message.bytes(), addr))?;
+            if let Err(e) = ready!(socket.poll_send_to(cx, message.bytes(), addr)) {
+                // Drop the UDP packet and continue
+                log::warn!("error sending message on udp_socket: {}", e);
```

Comment:
Can we add to this comment that we're dropping the response?

Reply:
Sure thing. Are you thinking something additional like "error sending message on udp_socket, dropping response: {}" ? I guess we can include the response itself as well if you think it would be relevant.

Reply:
Tweaked this a bit. Please feel free to reopen if you'd like it changed further.

Reply:
Sorry, didn't have a chance to respond. I think the ip-addr is useful, I guess the only additional thing is maybe add the `Display` version of the response Header (not the entire message) I'm good with this as is though.